docs-kits/kits/Eco_Pass_KIT/page-adoption-view.md

```diff
@@ -112,365 +112,590 @@ By providing a standardized, scalable and easy-to-adopt solution, the EcoPass KI
 
 ## Semantic Models
 
-Depending on the use case and related KIT, Catena-X provides different semantic models that help to structure and make use of data via semantic information. These models help to provide a basic meaning to the data and their relationship, thereby enabling interoperability between data sets. Catena-X data models rely on principles as understandability, standardization, accuracy, differentiation, auditability, comprehensiveness, and provision of insights to drive improvement actions.
-In its first version the KIT entails the data models Battery Pass and Transmission Pass, further data models will follow. The goal of these passports is to create unified data models that can be used and adapted across the automotive industry. These data models follow the Catena-X Standards [CX-0034](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_PDF_Maerz/Sustainability_Use_Case_Circular_Economy/CX_-_0034_Data_Model_BatteryPass_UseCase_CE_v_1.0.1.pdf), CX-0095 (insert once published) and are modeled according to the [CX-0003](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Archiv/Update_Juli_23_R_3.2/CX-0003-SAMMSemanticAspectMetaModel-v.1.0.2.pdf) BAMM Standard (BAMM has been renamed to SAMM thereafter) in Version 1.0.0 . For detailed information please follow the embedded links to the Catena-X Standard library.
+Depending on the use case and related KIT, Catena-X provides different semantic models that help to structure and make use of data via semantic information. These models help to provide a basic meaning to the data and their relationship, thereby enabling interoperability between data sets. Catena-X data models rely on principles as understandability, standardization, accuracy, differentiation, auditability, comprehensiveness, and provision of insights to drive improvement actions. The KIT entails the main data models of a Digital Product Passport and Battery Passport, whereat both are justified with an underlying regulation.
 
-### Battery Pass
+Other specific product passports which are further described in the EcoPass KIT are the [battery](#battery-passport), [transmission](#transmission-passport) and [electric drive](#electric-drive-passport). All of them depend on the generic Digital Product Passport data model. Further data models will follow.
 
-#### Battery Pass Introduction
+The goal of these passports is to create unified data models that can be used and adapted across the automotive industry. These data models are modelled according to the [CX-0003](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Archiv/Update_Juli_23_R_3.2/CX-0003-SAMMSemanticAspectMetaModel-v.1.0.2.pdf) BAMM/SAMM Standard in Version 1.1.0 (updated link once published). For detailed information please follow the embedded links to the Catena-X Standard library.
```

Comment:
As the link to the PDF-file of CX-0003 will be deprecated with the Release 24.05, I'd suggest to change the link to: https://catena-x.net/en/standard-library

PS: As BAMM models are deprecated with CX-0003:1.1 and only SAMM models are accepted in accordance to CX-0003, the part with "_BAMM/SAMM Standard_" should be changed in "_SAMM Standard_"

Reply:
Done